### DISCUSSION TITLE
-type gnuace not an option when trying to build using MPC

### DISCUSSION THREAD

**Discussion by mdavies2310** [OpenDDS/OpenDDS#4214] [Q&A] [answered]
Hi,
The type "gnuace" is not in the help menu and does not work for me. Is there something specific I need to do to enable this?

**Comment by jrw972** [comment]
Please share the your command you are using and its output.

**Reply by mdavies2310** [reply]
$MPC_ROOT/mwc.pl --type gnuace BITData.mwc
Output:
ERROR: Invalid type: gnuace
mwc.pl v4.1.46

Then it pops up the usage

**Reply by mitza-oci** [reply, marked as answer]
Don't use `$MPC_ROOT/` in front of `mwc.pl`.  The `mwc.pl` script that is required for this is already on the path (based on setenv/configure).

**Reply by mdavies2310** [reply]
Wow. Thats it! Thanks!

**Reply by mitza-oci** [reply]
If documentation was lacking for this please consider proposing a change (sources are all in the git repo).
